terraform/modules/happy-env-eks/dynamo.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_dynamodb_table" "locks" {
+  name           = "${var.tags.project}-${var.tags.env}-${var.tags.service}-stacklist"
+  billing_mode   = "PAY_PER_REQUEST"
+  read_capacity  = 5
+  write_capacity = 5
+  hash_key       = "key"
+
+  attribute {
+    name = "key"
+    type = "S"
+  }
+
+  server_side_encryption {
+    enabled = true
```

Comment:
Do we need encryption for the lock table?

Reply:
not sure I copied to from ECR